gstcefsrc.cc

```diff
@@ -632,7 +809,11 @@ gst_cef_src_stop (GstBaseSrc *base_src)
   GST_INFO_OBJECT (src, "Stopping");
 
   if (src->browser) {
-    src->browser->GetHost()->CloseBrowser(true);
+#ifdef __APPLE__
+    dispatch_async_f(dispatch_get_main_queue(), src, (dispatch_function_t)&gst_cef_src_close_browser);
+#else
+    gst_cef_src_close_browser(src);
```

Comment:
Here and elsewhere, on non-Apple does this have to be called from the event loop thread or can it be called from anywhere?

Reply:
On checking in my Mac (it's the only one where this can be life-or-death of CEF) it can be called from anywhere, CEF itself posts the event to the UI thread. I'll clean this up.